bin/check_versions

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+
+need_version=0
+
+for json_file in $(git diff --name-only master HEAD | grep '^exercises/.*\.json$' | sort --unique); do
```

Comment:
it occurs to me:

* the `sort`, although it does not affect correctness, may help us make sense of the output if the output is alphabetised
* the `uniq` or `--unique` neither affects correctness nor do I expect it to ever change the output. I cargo-culted from https://github.com/exercism/rust/blob/master/_test/ensure-readmes-are-updated.sh#L15 (there is a subtle pun there), but a `cut` operation is done there, whereas there is no `cut` here. If we assume that `git diff` only outputs each file once, then it is useless. I suppose it can be kept despite its uselessness, in case `git diff`'s behaviour ever changes, but I don't see why it ever would

Reply:
> the sort, although it does not affect correctness, may help us make sense of the output if the output is alphabetised

Since the output is directly from `git diff`, which will be pre-sorted, sort can be removed.

> the uniq or --unique neither affects correctness nor do I expect it to ever change the output.

Agreed. It is unnecessary and can also be removed.

Reply:
Upon consideration, I've decided that in this case "can be removed" implies "should be removed". If people assume things are there for a reason and try to find a reason for the `sort` and `uniq`, they will be at a loss, and I don't want to inflict that on others.